contracts/Receipt.sol

```diff
@@ -93,7 +93,7 @@ contract Receipt is IReceipt, Ownable {
         require(block.timestamp < receipt.releaseTime, "REC: claim time passed");
         require(!receipt.liquidatorRefundClaimed, "REC: Already claimed");
         // Validate the escrowed order was fully sold
-        (uint256 unitsSold, int256 avgPrice) = calcUnitsSold(orderIds, escrowId, market, receipt.time);
```

Comment:
wasn't mentioned in the PR, just wondering why `receipt.time` was removed?

Reply:
It was unused, because I just use receipt.time inside `calcUnitsSold`, so it's unnecessary